packages/cw1155/src/receiver.rs

```diff
@@ -0,0 +1,73 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::{to_binary, Binary, CosmosMsg, HumanAddr, StdResult, Uint128, WasmMsg};
+
+use crate::msg::TokenId;
+
+/// Cw1155ReceiveMsg should be de/serialized under `Receive()` variant in a HandleMsg
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub struct Cw1155ReceiveMsg {
+    pub operator: HumanAddr,
+    pub from: Option<HumanAddr>,
```

Comment:
Since this is an interface other devs will have to support, good to be explicit.

In this case `operator` was the one who actually executed the message and `from` is the account the funds were sent from, correct? If so, please just add some RustDoc strings here.

Reply:
Otherwise, nice job in this file.

Reply:
done